include/z64scene.h

```diff
@@ -456,9 +456,6 @@ typedef enum {
 #define SCENE_CMD_ACTOR_LIST(numActors, actorList) \
     { SCENE_CMD_ID_ACTOR_LIST, numActors, CMD_PTR(actorList) }
 
-#define SCENE_CMD_CAM_LIST(numCams, camList) \
-    { SCENE_CMD_ID_ACTOR_CUTSCENE_CAM_LIST, numCams, CMD_PTR(camList) }
```

Comment:
This didn't need to be removed, just replaced with a more accurate macro (see the original scene_header.h that I posted) on the discord

Reply:
If it is unused, why have a macro for it? Does it even works?

Reply:
It still has functionality so it should be supported even if functionally it does not do much

Reply:
From what I'm seeing, it only sets a pointer, but that pointer doesn't seem to be used in the whole repo to date.
Also, from reading the comments of that function, we know it was supposed to be a camera list for cutscenes ([here](https://github.com/zeldaret/oot/blob/0b8252cfe99651b3174ac8652ba133781e8e2105/src/code/z_scene.c#L208)), so if we will keep this macro, it should have an appropriate name, and maybe leave a comment saying it is unused and it functionality is non-implemented

Reply:
Well the thing is that we *don't* know. That information came from it's use case in majora's mask, but there is zero proof of it being intended to do that in oot which is why I don't not suggest naming it as such, especially when it is meant to be used in conjunction with command 1B which does not exist in OoT at all.

Reply:
If that the case, then we should remove that comment and change the name of `SCmdCsCameraList` and `csCameraList` to something more appropriate too.

Reply:
yes that needs to be changed, which you can do in this pr if you'd like, eventually I'm gonna do some docs on z_scene too

Reply:
Changed.